src/shared/icons/Play.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { s } from "./";
+
+export default function Play(props) {
+
+	const { colour } = props;
```

Comment:
```suggestion
	const { 
		colour, 
		className = "" 
	} = props;
```
Passing down a className allows us to scale the icon and add animations and stuff if we need to!